queenbee/plugin/plugin.py

```diff
@@ -186,6 +186,5 @@ def to_folder(self, folder_path: str, readme_string: str = None, license_string:
             with open(os.path.join(folder_path, 'README.md'), 'w') as f:
                 f.write(readme_string)
 
-        if license_string is not None:
-            with open(os.path.join(folder_path, 'LICENSE'), 'w') as f:
-                f.write(license_string)
+        with open(os.path.join(folder_path, 'LICENSE'), 'w') as f:
```

Comment:
Should we just drop the LICENSE file if we're not actually going to write anything there?

Reply:
I think it might still be helpful for user discovery. We can remove it at any point and it shouldn't break anything since it is only generated insider the folder when one uses to_folder.